src/fsharp/FSharp.Core/list.fs

```diff
@@ -223,7 +223,7 @@ namespace Microsoft.FSharp.Collections
 
         [<CompiledName("Pairwise")>]
         let pairwise (list: 'T list) =
-            Microsoft.FSharp.Primitives.Basics.List.pairwise list
```

Comment:
@dsyme Yea I made this commit so that the `List` related PRs I did were in line with the previously completed cleanups by @forki.